### DISCUSSION TITLE
IM is not generating proper thumbnail and preview for DN files

### DISCUSSION THREAD

**Discussion by SAN2INGIT** [ImageMagick/ImageMagick#5098] [Help] [answered]
I have a DNG image ![sample1crop](https://user-images.githubusercontent.com/32328926/165892958-0363dba0-e753-4eba-af4d-873e3bacefd9.jpg), edited by adobe lightroom. 

The DNG file has some metadata:
1.  EXIF:IFD0:{"PreviewColorSpace": "Gray Gamma 2.2"}
2.  XMP:XMP-crs:{
     "CropTop": 0.074189,
    "CropLeft": 0.056643,
    "CropBottom": 0.64915,
    "CropRight": 0.656025,
    "CropAngle": 0,
    "CropConstrainToWarp": 0,
    "HasCrop": true,
}

The generated preview and thumbnail image is not honoring these metadata, thus generating a wrong thumbnail and preview. 
IM commands used to generate the thumbnail:
`magick.exe C:\sample1crop.dng -resize 500x375 C:\crop-T.JPG`
or
`magick.exe dng:C:\sample1crop.dng -intent relative -sample 500x375> -strip -auto-orient -density 72 C:\crop-T.JPG`
or
`magick.exe convert dng:C:\sample1crop.dng -thumbnail 500x375 -filter -auto-orient -density 72 C:\orig-T.JPG`

My question is: Does IM has the capability to honor these metadata while resizing?

**Comment by snibgo** [comment, marked as answer]
> My question is: Does IM has the capability to honor these metadata while resizing?

No, it doesn't.

Different manufacturers record such metadata in different ways. So any software that tries to honour cropping metadata would need to do different processing for all the manufacturers, and often different models of cameras or software within each manufacturer. 

That could be done, in theory. Perhaps some specialist raw processing programs (eg darktable, RawTherapee, Adobe Lightroom) do that.

IM uses libraw to process DNG files. If libraw is given the capability to honour that metadata, IM could then also do so.
